scripts/Muon/GUI/Common/home_runinfo_widget/home_runinfo_widget_presenter.py

```diff
@@ -0,0 +1,40 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2018 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+from __future__ import (absolute_import, division, print_function)
+
+from Muon.GUI.Common.home_tab.home_tab_presenter import HomeTabSubWidget
+
+
+class HomeRunInfoWidgetPresenter(HomeTabSubWidget):
+
+    def __init__(self, view, model):
+        self._view = view
+        self._model = model
+
+    def show(self):
+        self._view.show()
+
+    def update_view_from_model(self):
+        self._view.clear()
+        run = self._model.get_run_number()
+        instrument = self._model.get_instrument_name()
+        self._view.add_text_line("Instrument                : " + str(instrument))
+        self._view.add_text_line("Run                       : " + run)
+        self._view.add_text_line(self.create_text_line("Title                    ", "run_title"))
+        self._view.add_text_line("Comment                   : " + str(self._model.get_workspace_comment()))
+        self._view.add_text_line(self.create_text_line("Start                    ", "run_start"))
+        self._view.add_text_line(self.create_text_line("End                      ", "run_end"))
+        self._view.add_text_line(self.create_text_line("Good Frames              ", "goodfrm"))
+        self._view.add_text_line("Counts (MeV)              : " + str(self._model.get_counts_in_MeV()))
```

Comment:
put counts before good frames and add `counts/good frames`. Will need to check if good frames are available (they are not always)